File: api/src/main/java/gg/mineral/api/world/property/LevelType.java
package gg.mineral.api.world.property;

import lombok.RequiredArgsConstructor;

@RequiredArgsConstructor
public enum LevelType {
    DEFAULT("default"), FLAT("flat"), LARGE_BIOMES("largeBiomes"), AMPLIFIED("amplified"), DEFAULT_1_1("default_1_1");

    private final String type;

    public String string() {
        return type;
    }
}

File: api/src/main/java/gg/mineral/api/world/property/LevelType.kt
package gg.mineral.api.world.property

import lombok.RequiredArgsConstructor

@RequiredArgsConstructor
enum class LevelType {
    DEFAULT("default"), FLAT("flat"), LARGE_BIOMES("largeBiomes"), AMPLIFIED("amplified"), DEFAULT_1_1("default_1_1");

    private val type: String? = null

    fun string(): String? {
        return type
    }
}
